packs/hubot/rules/notify_hubot.yaml

```diff
@@ -7,7 +7,7 @@ trigger:
   pack: "chatops"
   type: "core.st2.generic.notifytrigger"
 criteria:
-  trigger.channel:
+  trigger.routes:
```

Comment:
Good question. I'm pretty sure it's singular `route`, since this was nothing more than a rename.